src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
-body {
-  margin: 0;
+@import "./utils/reset";
+@import "./utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap};
+  width: $calendar-width;
+}
+
+.calendar__day {
+  box-sizing: border-box;
+  width: #{$cell-size};
+  height: #{$cell-size};
+  background-color: #eee;
+  border: #{$border-width} solid #{$border-color};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: $font-family;
+  font-size: 30px;
+  font-weight: bold;
+  position: relative;
+  cursor: pointer;
+
+  &:hover {
+    background-color: pink;
+    transform: translateY(-20px);
+    transition: transform 0.5s ease;
```

Comment:
transition related properties should ALWAYS be applied on element itself, not on `:hover`